test/test_jit.py

```diff
@@ -10032,6 +10032,12 @@ def forward(self, x, skip_name):
             self.checkModule(M(), (inp, name))
             self.checkModule(M2(), (inp, name))
 
+    def test_list_keyword(self):
+        def foo():
```

Comment:
Can you add a check for a bad input as well? e.g. that `list(1)` says something like Python's `TypeError: 'int' object is not iterable`

Reply:
there are tests that already do that like `test_sum_list_wrong_type`